diff -r 4178904d7698 src/CMakeLists.txt
--- a/src/CMakeLists.txt	Sat Dec 11 16:35:55 2021 -0500
+++ b/src/CMakeLists.txt	Fri Dec 01 13:00:32 2023 -0300
@@ -643,13 +643,6 @@
        add_definitions( -DBACKPATCH )
 endif( BACKPATCH )

-# Update gitinfo.h
-
-add_custom_target( revision_check ALL
-       COMMAND updaterevision src/gitinfo.h
-       WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-       DEPENDS updaterevision )
-
 # Libraries ZDoom needs

 message( STATUS "Fluid synth libs: ${FLUIDSYNTH_LIBRARIES}" )
@@ -1474,7 +1467,7 @@
        ${CMAKE_BINARY_DIR}/gdtoa
        ${SYSTEM_SOURCES_DIR} )

-add_dependencies( zdoom revision_check )
+#add_dependencies( zdoom revision_check )
 # [TP]
 add_dependencies( zdoom protocolspec )
