diff --git a/src/core/pathfinder/pathfinder.cpp b/src/core/pathfinder/pathfinder.cpp
index 9b363613..9862486b 100644
--- a/src/core/pathfinder/pathfinder.cpp
+++ b/src/core/pathfinder/pathfinder.cpp
@@ -137,9 +137,7 @@ PathFinder PathFinder::genericPathFinder(const QStringList &suffixes)
 		<< DataPaths::programFilesDirectory(DataPaths::x64)
 		<< DataPaths::programFilesDirectory(DataPaths::x86);
 	#else
-	paths << "/usr/bin" << "/usr/local/bin" << "/usr/share/bin"
-		<< "/usr/games/" << "/usr/local/games/"
-		<< "/usr/share/games/" << QCoreApplication::applicationDirPath() << ".";
+	paths << gDefaultDataPaths->workingDirectory() << ".";
 	#endif
 	QStringList pathsCopy(paths);
 	for (const QString &path : pathsCopy)
diff --git a/src/core/pathfinder/wadpathfinder.cpp b/src/core/pathfinder/wadpathfinder.cpp
index 07df0b64..6300542d 100644
--- a/src/core/pathfinder/wadpathfinder.cpp
+++ b/src/core/pathfinder/wadpathfinder.cpp
@@ -84,10 +84,6 @@ public:
 	QStringList defaultPaths()^M
 	{^M
 		QStringList paths;^M
-		#ifdef Q_OS_UNIX^M
-		paths << "/usr/local/share/games/doom/"^M
-			<< "/usr/share/games/doom/";^M
-		#endif^M
 		return paths;^M
 	}^M
 };^M

